File: engine/model/src/main/scala/tools/packet/guild/CreateGuildMessage.scala
package tools.packet.guild

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class CreateGuildMessage(private var _masterCharacterName: String, private var _guildName: String) extends PacketInput {
  def masterCharacterName: String = _masterCharacterName

  def guildName: String = _guildName

  override def opcode(): SendOpcode = SendOpcode.GUILD_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/guild/CreateGuildMessage.java
package tools.packet.guild;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record CreateGuildMessage(String masterCharacterName, String guildName) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.GUILD_OPERATION;
   }
}